### PR TITLE
Feature/5659 add metadata link

### PR DESCRIPTION
**Pull request by HavierD**

### CODE REVIEW

File: src/pages/detail-page/subpages/components/ContactBlock.tsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ const ContactBlock: React.FC<ContactBlockProps> = ({ title, contacts }) => {
           key={index}
           title={contact.organization + suffix}
           isContactFragment
-          email={contact.emails[0]}
+          email={contact.emails ? contact.emails[0] : undefined}
```
Comment (vietnguyengit) [inline src/pages/detail-page/subpages/components/ContactBlock.tsx:22]: no big deal, but what if `contact.emails[0] = ""`?

File: src/pages/detail-page/subpages/tab-panels/MetadataInformationPanel.tsx [hunk 2]
```diff
@@ -70,7 +89,7 @@ const MetadataInformationPanel = () => {
       },
       {
         title: "Full Metadata Link",
-        component: <div>full metadata link</div>,
+        component: <PlainLinkBlock url={metadataLink} />,
```
Comment (vietnguyengit) [inline src/pages/detail-page/subpages/tab-panels/MetadataInformationPanel.tsx:92]: so why dont you call `PlainLinkBlock` to be `MetadataLinkBlock`? unless `plain link block` is something called by STAC, I'd suggest a different name for clarity. "Plain" can be replaced by something else.

File: src/pages/detail-page/subpages/components/ContactBlock.tsx (hunk 1 above)
Comment (HavierD) [reply src/pages/detail-page/subpages/components/ContactBlock.tsx:22]: If any details is falsy("", undefined, null, etc...), the corresponding area will show something like "no *** data". Steve has this design, but it needed to change according to their discussion. So I will implement all the falsy data when the design is approved. I will then raise a ticket for it (will paste the link here shortly)

Comment (HavierD) [reply src/pages/detail-page/subpages/components/ContactBlock.tsx:22]: https://github.com/aodn/backlog/issues/5780

File: src/pages/detail-page/subpages/components/PlainLinkBlock.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import React, { ReactNode } from "react";
+import PlainTextFragment from "./PlainTextFragment";
+import BlockList from "./BlockList";
+import { Link } from "@mui/material";
+
+interface PlainLinkBlockProps {
```
Comment (vietnguyengit) [inline src/pages/detail-page/subpages/components/PlainLinkBlock.tsx:6]: rename @HavierD ?

File: src/components/list/LinkList.tsx [hunk 4]
```diff
@@ -3,11 +3,11 @@ import { Link } from "@mui/material";
 import TextItem from "./listItem/TextItem";
 import ExpandableList from "./ExpandableList";
 
-interface PlainLinkBlockProps {
+interface LinkListProps {
   url: string;
 }
 
-const PlainLinkBlock: React.FC<PlainLinkBlockProps> = ({ url }) => {
+const LinkList: React.FC<LinkListProps> = ({ url }) => {
```
Comment (vietnguyengit) [inline src/components/list/LinkList.tsx:10]: this name can be confusing with https://en.wikipedia.org/wiki/Linked_list

File: src/components/list/LinkList.tsx [hunk 5]
```diff
@@ -20,4 +20,4 @@ const PlainLinkBlock: React.FC<PlainLinkBlockProps> = ({ url }) => {
   return <ExpandableList title={"Full Metadata Link"} childrenList={link} />;
```
Comment (vietnguyengit) [inline src/components/list/LinkList.tsx:20]: you are hard-coded "metadata link" here, so this component is for list of "metadata link" only?

File: src/components/list/LinkList.tsx (hunk 4 above)
Comment (HavierD) [reply src/components/list/LinkList.tsx:10]: oh, sorry. Let me change another one

**Review by vietnguyengit — APPROVED** [review]

**Pull request merged**